app/src/main/kotlin/at/bitfire/davdroid/servicedetection/CollectionListRefresher.kt

```diff
@@ -418,4 +421,11 @@ class CollectionListRefresher @AssistedInject constructor(
                 false
         }
     }
+
+    private fun isHomeSetPersonal(davResponse: Response): Boolean {
+        // Owner must be set in order to check if the home set is personal
+        val ownerHref = davResponse[Owner::class.java]?.href?.toUri() ?: return false
+        // If Owner is set, check if the paths match. Owner is relative, doesn't contain the full URL, so we have to partially check it
+        return ownerHref.encodedPath == service.principal?.encodedPath
```

Comment:
We should not only compare the path. There's `HttpUrl.equalsForWebDAV` in `UrlUtils`.